src/c/examples/random/res/configuration.toml

```diff
@@ -10,6 +10,8 @@
   RequestTimeout = '5s'
   StartupMsg = 'Example random device started'
   HealthCheckInterval = '10s'
+  Host = ''	# Insert docker hosted ip address here
```

Comment:
Service will not start with Host set to an empty string, please remove this line

Reply:
Thought of putting a comment to update the Host address as per the docker hosting
Removed by the way
Please check